script/script_hooks/bootstrap

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+install_swift_format ()
```

Comment:
- This script should be in `script/common`. Just copy the format for `install_swiftlint`/`swiftlint` or `install_tailor`/`tailor`
- This must be called from `bootstrap`.
- This should be in https://github.com/guidomb/ios-scripts and once the PR is approved, update the scripts here.